File: src/person/Person.java
package person;

//Kaan Camci 150119063
public abstract class Person {

	protected String firstName;
	protected String lastName;

	public Person(String firstName, String lastName) {
		this.firstName = firstName;
		this.lastName = lastName;
	}

	// Creating get and set methods for variables
	public String getFirstName() {
		return firstName;
	}

	public void setFirstName(String firstName) {
		this.firstName = firstName;
	}

	public String getLastName() {
		return lastName;
	}

	public void setLastName(String lastName) {
		this.lastName = lastName;
	}

	public String getFullName() {
		return this.firstName + " " + this.lastName;
	}

	abstract public String getID();

}

File: pythonCode/Person.py
from abc import ABC , abstractmethod
class Person():

    def __init__(self):
        pass

    #  Creating get and set methods for variables
    def getFirstName(self):
        return self.__firstName

    def setFirstName(self, firstName:str):
        self.__firstName = firstName

    def getLastName(self):
        return self.__lastName

    def setLastName(self, lastName : str):
        self.__lastName = lastName

    def getFullName(self):
        return self.__firstName + " " + self.__lastName

    @abstractmethod
    def getID(self): str